micropy/project/template.py

```diff
@@ -219,6 +219,10 @@ def context(self):
             paths = list(self.iter_relative_paths(self.paths, strict=True))
         if self.local_paths:
             paths = [*list(self.iter_relative_paths(self.local_paths)), *paths]
+        # automatically add src/lib if it exists
```

Comment:
Why only if src/lib exists ?
if i run init a 2nd time , will it then pick-up on this , or do I need to 
- create a project 
- create src\lib 
- run microcli init

Reply:
Hmm... that's a good point. Guess it doesn't really matter if it doesn't exist anyways. Will fix this and finish this PR up tonight.